File: packages/cli/templates/project/function-stream-trigger.ts
import type { DataStreamsEvent, InvocationContext } from "@vibecloud/function-trigger-datastream";

export async function {{HANDLER}}(
  event: DataStreamsEvent,
  context: InvocationContext,
): Promise<void> {
  throw new Error(
    `Data Streams handler is not implemented (${event.messages.length} messages, request ${context.requestId})`,
  );
}

File: packages/cli/templates/project/function-stream-trigger.py
from typing import Any


def {{HANDLER}}(event: dict[str, Any], context: Any) -> None:
    message_count = len(event.get("messages", []))
    request_id = getattr(context, "request_id", "unknown")
    raise NotImplementedError(
        f"Data Streams handler is not implemented ({message_count} messages, request {request_id})"
    )
